radio/src/datastructs_private.h

```diff
@@ -995,8 +996,6 @@ PACK(struct RadioData {
 #elif LCD_W == 128
   uint8_t invertLCD:1;          // Invert B&W LCD display
   NOBACKUP(uint8_t spare:3 SKIP);
-#else
-  NOBACKUP(uint8_t spare:4 SKIP);
```

Comment:
Why has this been deleted? Removing it breaks the alignment.

Reply:
Thanks for pointing that out. Not intentional. My excuse: fat fingers.

I'll fix that with together with the missing translation